app/src/main/java/com/example/ubclaunchpad/launchnote/photoBrowser/AllPhotosFragment.kt

```diff
@@ -75,6 +80,45 @@ class AllPhotosFragment : Fragment() {
         }
     }
 
+    fun saveSelection() {
+                Flowable.fromCallable {
```

Comment:
Please make the indents 2 spaces instead of 4, and add some comments for documentation to explain what you did.